server/src/test/java/org/elasticsearch/cluster/metadata/AliasMetaDataTests.java

```diff
@@ -95,6 +111,7 @@ private static AliasMetaData createTestItem() {
         if (randomBoolean()) {
             builder.filter("{\"term\":{\"year\":2016}}");
         }
+        builder.writeIndex(randomBoolean());
```

Comment:
sometime make it null?

Reply:
I think it is ok to leave it as static since that is the nature of this test.

I believe testToAndFromXContent captures the random scenario, and so I will update it so
that randomAlias includes the write index randomization